modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/content/Language_zh_TW.properties

```diff
@@ -15,7 +15,10 @@ denylist-name=Deny List
 editors-name=啟用編輯器分析
```

Comment:
We should split the result of calling buildLang into their own commit to simplify review/rebasing and allowing Brian to remove it if it wants to call it with integration with the translation tool

Reply:
Yes, it makes sense, I'll do it.